src/Components/CreateImageWizardV2/steps/Review/ReviewStepTextLists.tsx

```diff
@@ -525,8 +531,56 @@ export const OscapList = () => {
         <TextListItem component={TextListItemVariants.dd}>
           {oscapProfile}
         </TextListItem>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
+          Kernel arguments:
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          <CodeBlock>
+            <CodeBlockCode id="code-content">
+              {data?.kernel?.append}
+            </CodeBlockCode>
+          </CodeBlock>
+        </TextListItem>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
+          Disabled services:
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          <CodeBlock>
+            <CodeBlockCode id="code-content">
+              {disableServicesDisplayString}
+            </CodeBlockCode>
+          </CodeBlock>
+        </TextListItem>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
+          Enabled services:
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          <CodeBlock>
+            <CodeBlockCode id="code-content">
+              {enabledServicesDisplayString}
+            </CodeBlockCode>
+          </CodeBlock>
+        </TextListItem>
+        <Alert
+          variant="info"
+          isInline
+          isPlain
+          title="Additional customizations"
+        >
+          Selecting an OpenSCAP profile will cause the appropriate packages,
+          file system configuration, kernel arguments, and services to be added
+          to your image.
+        </Alert>
```

Comment:
The Alert will need to be put outside of the TextList otherwise the rendering is chaotic. See https://github.com/osbuild/image-builder-frontend/pull/1611#issuecomment-1933752350

![image](https://github.com/osbuild/image-builder-frontend/assets/49452678/6a337260-bc21-46c5-8d09-d3cfa3112e1c)